src/db/helper.ts

```diff
@@ -0,0 +1,5 @@
+import { settings } from '../settings';
```

Comment:
I think you're creating a new client each time you import this file. Perhaps you can use a singleton pattern: https://blog.bitsrc.io/the-singleton-pattern-in-typescript-b906303fda93